pandas/core/ops/array_ops.py

```diff
@@ -161,7 +161,7 @@ def arithmetic_op(
     right: Any,
     op,
     str_rep: str,
-    eval_kwargs: Dict[str, str],
+    eval_kwargs: Dict[str, bool],
```

Comment:
Nice this makes sense. Would certainly love a follow up to comment that all the way through